### DISCUSSION TITLE
Failed to compile with zksolc: Cannot get source files

### DISCUSSION THREAD

**Discussion by zhangninghai** [zkSync-Community-Hub/zksync-developers#477] [Compilers]
### Which compiler are you using?

- [X] zksolc
- [ ] zkvyper

### Which version of Solidity or Vyper?

0.8.24

### zksolc/zkvyper version?

0.8.24

### Explain the issue

```bash
...
~/Documents/code/vscode/zksync/test (main*) » forge build --zksync                                     zhangninghai@zhangninghaideMacBook-Pro
[⠰] Compiling...
No files changed, compilation skipped
Error: 
Failed to compile with zksolc: Cannot get source files
```

**Comment by dutterbutter** [comment]
@zhangninghai Do you have a link to the repo you are trying to compile?

**Comment by nooma-42** [comment]
Did you put it in the $PATH?

**Comment by dutterbutter** [comment]
Important to note we are making a large update to the compiling process in foundry-zksync very soon. I am going to close this due to inactivity. Please feel free to re-open on the `foundry-zksync` repo.
